plugins/users/settings/delete.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const boom = require('@hapi/boom');
+const schema = require('screwdriver-data-schema');
+
+module.exports = () => ({
+    method: 'DELETE',
+    path: '/users/settings',
+    options: {
+        description: 'Reset user settings',
+        notes: 'Reset user settings',
+        tags: ['api', 'users'],
+        auth: {
+            strategies: ['token'],
+            scope: ['user', '!guest']
+        },
+
+        handler: async (request, h) => {
+            const { userFactory } = request.server.app;
+            const { scmContext, username } = request.auth.credentials;
+            const { settings } = request.payload;
```

Comment:
This is a `delete`, it expects no payload

Reply:
Okay, found an example. Please refer, https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/pipelines/caches/delete.js

Reply:
Actually, here's another example https://github.com/screwdriver-cd/screwdriver/blob/master/plugins/buildClusters/remove.js